test/salt/pillar/rsyncd.sls

```diff
@@ -0,0 +1,13 @@
+rsyncd:
```

Comment:
If you accept https://github.com/saltstack-formulas/rsyncd-formula/pull/2/files#r335246555, then this file will obviously need to be renamed.